### PR TITLE
MEN-5124: adjust tests to the new email sender in workflows-enterprise

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Comment by tranchitella** [comment]
Combined QA pipeline: https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/379384822

File: tests/tests/test_monitor_client.py [hunk 1]
```diff
@@ -274,7 +274,7 @@ def test_monitorclient_alert_email(self, monitor_commercial_setup_no_client):
         """Tests the monitor client email alerting"""
         mailbox_path = "/var/spool/mail/local"
         wait_for_alert_interval_s = 8
-        expected_from = "noreply@mender.io"
+        expected_from = "no-reply@hosted.mender.io"
```
Comment (merlin-northern) [inline tests/tests/test_monitor_client.py:277]: `noreply@mender.io"` was something that was agreed with Eystein, I know these are only tests -- but do we have a good reason not to keep noreply@mender.io here?

Comment (tranchitella) [reply tests/tests/test_monitor_client.py:277]: Because of SPF/DMARC/DKIM settings, we cannot send emails for `@mender.io` from a non-Google SMTP server. That's why we have to move to `@hosted.mender.io`. We can keep whatever we want in the test, but I tried to keep it as close as possible to the reality.

Comment (merlin-northern) [reply tests/tests/test_monitor_client.py:277]: exactly my point to keep it close to reality :) if we now cannot send from @medner.io, then it is fine. thanks. I will let Eystein know.

File: docker-compose.override.os.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+version: '2.1'
+services:
+    mender-gui:
+        volumes:
+            - ../gui/dist:/var/www/mender-gui/dist
```
Comment (merlin-northern) [inline docker-compose.override.os.yml:5]: why is that needed? probably should be a separate commit?

Comment (tranchitella) [reply docker-compose.override.os.yml:5]: oh sorry, this is a local file which wasn't meant to be added. I'm removing it.

**Comment by tranchitella** [comment]
I won't rerun the pipeline, the force-commit is because of an extra file which was added by mistake (`docker-compose.override.os.yml`).

**Review by merlin-northern — APPROVED** [review]
lgtm
